.chloggen/ottl_statements_context_change_grammar.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

```suggestion
change_type: breaking
```

Since we are adding a method to the interface we need to mark this as breaking.

---

Oh that's true, good catch!